<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseSet</class>
 <widget class="QDialog" name="BaseSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基座设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>基座1：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Base1_comboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>基座2：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Base2_comboBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>基座3：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Base3_comboBox"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButtonConfirm">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
